GoogleChromeLabs/ps-analysis-tool
# Fix: Retention period cookie filter

## Description
This PR adds code to update the default value returned by `calculateEffectiveExpiryDate` to `Session`. 
And take care of assignments/processing of required files.
- Because of the above change, the retention period filter can properly filter out rows due to consistent values.


## Relevant Technical Choices



## Testing Instructions
- Open the PSAT panel, and navigate to the first frame.
- Open the filters sidebar and open the retention period filter.
- Select Session filter value, the rows showing data should have Session value in the expires column.


## Additional Information:



## Screenshot/Screencast


https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/771e56cf-bb8b-4512-bdc7-ff03bc7baede





---

## Checklist






Fixes #
